### PR TITLE
Fastx toolkit 0.0.14 goolf 1.7.20

### PR DESCRIPTION
**Pull request by npklein**
depends on #1706

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/f/FASTX-Toolkit/FASTX-Toolkit-0.0.14-goolf-1.7.20.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Cedric Laczny <cedric.laczny@uni.lu>, Fotis Georgatos <fotis@cern.ch>
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/f/FASTX-Toolkit/FASTX-Toolkit-0.0.14-goolf-1.7.20.eb]: I suggest to add an extra Authors:: line with your name, if you prefer.
This permits to address a bunch of interested/related people easily (git shows only committers btw).

File: easybuild/easyconfigs/f/FASTX-Toolkit/FASTX-Toolkit-0.0.14-goolf-1.7.20.eb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        ['bin/fastx_%s' % x for x in 
+            ['clipper', 'trimmer', 'quality_stats', 'artifacts_filter', 'reverse_complement',
+            'collapser', 'uncollapser', 'renamer', 'barcode_splitter.pl', 'nucleotide_distribution_graph.sh',
+            'nucleotide_distribution_line_graph.sh']] + 
+        ['bin/fasta_%s' % x for x in 
+            ['clipping_histogram.pl', 'formatter', 'nucleotide_changer']] +
+        ['bin/fastq_%s' % x for x in 
+            ['quality_boxplot_graph.sh', 'quality_converter', 'to_fasta', 'quality_filter', 
+            'quality_trimmer', 'masker']],
+    'dirs': ['.']
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/f/FASTX-Toolkit/FASTX-Toolkit-0.0.14-goolf-1.7.20.eb]: oops, these `'.'` thingies are left-overs from (my?) hackish practice some time ago; better drop/empty that.

**Comment by migueldiascosta** [comment]
@npklein, this PR is being closed for the following reason(s): no activity for > 1 year.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**